draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -1380,13 +1380,15 @@ Transfer-Encoding: gzip, chunked
   <x:ref>trailer-section</x:ref>   = *( <x:ref>field-line</x:ref> <x:ref>CRLF</x:ref> )
 </sourcecode>
 <t>
-   A recipient that decodes and removes the chunked encoding from a message
-   (e.g., for storage or forwarding to a non-HTTP/1.1 peer) &MUST; discard
-   any received trailer fields, store/forward them separately from the header
-   fields, or selectively merge into the header section only those trailer
-   fields corresponding to header field definitions that are understood by
-   the recipient to explicitly permit and define how their corresponding
-   trailer field value can be safely merged.
+   A recipient that removes the chunked encoding from a message &MAY;
+   selectively retain or discard the received trailer fields. A recipient
+   that retains a received trailer field &MUST; either store/forward the
+   trailer field separately from the received header fields (e.g., within a
+   separate table or trailer section) or merge the received trailer field
```

Comment:
separate "table"?

Reply:
yes, that's one example

Reply:
they can be stored as a separate lookup table or forwarded as a separate trailer section

Reply:
easier to just remove the example